### PR TITLE
Update through all scripts

### PR DESCRIPTION
**Pull request by vladimirbazhanov**
- Updated OS X image to use XCode 9.2
- added 'readline' installation before Ruby compilation to prevent problems with cyrillic letters in irb console
- changed version of Qt to 5.5, the last version that can be used to correctly compile capybara-webkit
- Added Skype
- Added Slack
- removed Hipchat, Dropbox, Firefox and Dash
- removed osxlockdown, because it was not working and not maintained anymore
- fixed some minor syntax error in bash scripts
- Installed Ruby versions is 2.5.0 now
- tmuxinator is not installed anymore

### CODE REVIEW

File: modules/ruby.bash [hunk 1]
```diff
@@ -8,9 +8,9 @@ eval "$(rbenv init -)"
 
 info_echo "Set default gems list"
 echo "bundler" >> "$(brew --prefix rbenv)/default-gems"
-echo "tmuxinator" >> "$(brew --prefix rbenv)/default-gems"
```
Comment (dmitrytrager) [inline modules/ruby.bash]: *feels sad*

File: modules/brew.bash [hunk 2]
```diff
@@ -15,12 +15,18 @@ info_echo "Install Brew formalue"
 brew tap "Homebrew/bundle" 2> /dev/null
 brew bundle --file="$osx_bootstrap/Brewfile"
 
+# Linking Qt to have 'qmake' for building Capybara
```
Comment (timurvafin) [inline modules/brew.bash]: Let's add link to Capybara install instructions

File: modules/brew.bash [hunk 3]
```diff
@@ -15,12 +15,18 @@ info_echo "Install Brew formalue"
...
+
 # https://github.com/eventmachine/eventmachine/issues/602#issuecomment-152184551
 info_echo "Link keg-only openssl to /usr/local to let software outside of Homebrew to find it"
 brew unlink openssl && brew link openssl --force
 
 info_echo "Link Curl with openssl"
 brew link --force curl
 
+info_echo "Link readline to correctly build Ruby later"
+brew link --force readline
```
Comment (timurvafin) [inline modules/brew.bash]: Could we try to link these formulae in the [`Brewfile`](https://github.com/Homebrew/homebrew-bundle#usage)

File: .travis.yml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 os: osx
-osx_image: xcode7.3
+osx_image: xcode9.2
```
Comment (timurvafin) [inline .travis.yml]: Could we test on `OS X 10.13`?

File: modules/ruby.bash [hunk 5]
```diff
@@ -8,9 +8,9 @@ eval "$(rbenv init -)"
...
 
-ruby_version="2.3.1"
+info_echo "Installing latest stable Ruby..."
+ruby_version="2.5.0"
```
Comment (timurvafin) [inline modules/ruby.bash]: `ver="$(rbenv install -l | grep -E "^\s*([.0-9]+)$" | tail -1)"`

**Comment by timurvafin** [comment]
Please make sure we have no error in the build https://travis-ci.org/fs/osx-bootstrap/builds/355994960#L1821

**Pull request merged**